test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -645,3 +646,227 @@
+                               :character_maximum_length nil
+                               :numeric_precision nil
+                               :numeric_precision_radix nil
+                               :data_type "bytea"
+                               :column_name "value_hash"
+                               :table_name "fact_values"}
+                              :right-only nil
+                              :same nil}
+                             {:left-only
+                              {:numeric_scale 0
+                               :column_default nil
+                               :character_octet_length nil
+                               :datetime_precision nil
+                               :nullable? "NO"
+                               :character_maximum_length nil
+                               :numeric_precision 64
+                               :numeric_precision_radix 2
+                               :data_type "bigint"
+                               :column_name "fact_value_id"
+                               :table_name "facts"}
+                              :right-only nil
+                              :same nil}
+                             {:left-only
+                              {:numeric_scale 0
+                               :column_default "nextval('fact_values_id_seq'::regclass)"
+                               :character_octet_length nil
+                               :datetime_precision nil
+                               :nullable? "NO"
+                               :character_maximum_length nil
+                               :numeric_precision 64
+                               :numeric_precision_radix 2
+                               :data_type "bigint"
+                               :column_name "id"
+                               :table_name "fact_values"}
+                              :right-only nil
+                              :same nil}
+                             ;; new columns
+                             {:left-only nil
+                              :right-only
+                              {:numeric_scale nil
+                               :column_default nil
+                               :character_octet_length nil
+                               :datetime_precision nil
+                               :nullable? "YES"
+                               :character_maximum_length nil
+                               :numeric_precision nil
+                               :numeric_precision_radix nil
+                               :data_type "bytea"
+                               :column_name "large_value_hash"
+                               :table_name "facts"}
+                              :same nil}
+                             ;; shrank value_type_id from bigint to integer
+                             {:left-only {:data_type "bigint" :numeric_precision 64}
+                              :right-only {:data_type "integer" :numeric_precision 32}
```

Comment:
After a bit of discussion with @senior, I think we should back this out and handle it later, separately, if it still seems worth pursuing, and we should probably fix the value_types.id mismatch then too.